src/ui/mainwindow.cc

```diff
@@ -741,11 +741,7 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
 
   scanPopup->setStyleSheet( styleSheet() );
 
-  connect( scanPopup,
-           SIGNAL( editGroupRequested( unsigned ) ),
-           this,
-           SLOT( editDictionaries( unsigned ) ),
-           Qt::QueuedConnection );
+  connect( scanPopup, &ScanPopup::editGroupRequest, this, &MainWindow::editDictionaries, Qt::QueuedConnection );
 
```

Comment:
can the last parameter `Qt::QueuedConnection` be removed?

Reply:
https://doc.qt.io/qt-6/qt.html#ConnectionType-enum

I don't see any difference between `DirectConnection` and `QueuedConnection` here (and many other connects in mainwindow.cc) because both sender and receiver live in the same thread :sweat_smile:

Reply:
There are multiple QueuedConnection in mainwindow.cc need some investigation to know if they are pointless.

Maybe do it in one PR.